apps/src/lab2/views/Lab2Wrapper.module.scss

```diff
@@ -2,7 +2,7 @@
 @import './common.scss';
 
 .labContainer {
-  position: fixed;
+  position: absolute;
   top: $top-spacing;
```

Comment:
Looking at this, it's probably more ideal to not use absolute positioning at all, and let the page lay us out wherever we need to be so we don't need to make any assumptions about the size of the header - but that's for a separate change 🙂